R/helperFunctions.R

```diff
@@ -128,8 +128,18 @@ getId <- function(x) reg_match0(x, '(?<= Id=")[0-9A-Za-z]+')
 #' @keywords internal
 #' @noRd
 validateColour <- function(colour, errorMsg = "Invalid colour!") {
+  colour <- check_valid_colour(colour)
```

Comment:
Oh I always wanted to get rid of the `colours` in favor of `colors`

Reply:
I've very prone to using `colours`.  Might be worth it to go through all the code/documentation and use just `colors` for everything.